package.json

```diff
@@ -11,11 +11,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^4.0.1",
+    "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "express-graphql": "^0.9.0",
     "graphql": "^14.6.0",
+    "graphql-upload": "^10.0.0",
+    "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.15",
-    "mongoose": "^5.9.3"
+    "mongoose": "^5.9.3",
+    "react-native-dotenv": "^0.2.0"
```

Comment:
don't think you need this dependency here right?

Reply:
You have the eyes of an eagle! Yes, not needed. Used it before but then removed it